### ISSUE TITLE
You're unable to play because you may be offline | LoL

### ISSUE THREAD

**Issue by stargamer57** [molenzwiebel/Deceive#212]
![ApplicationFrameHost_VEKr](https://github.com/molenzwiebel/Deceive/assets/86321456/5680ca63-1326-4d86-b072-91069eeece33)
Did Riot implement this new thing? I've never had this problem before

**Comment by molenzwiebel** [comment]
I need some more information on this. Does it consistently happen, and only with Deceive?

**Comment by stargamer57** [comment]
yes it only happens when i have deceive running. after i go online on deceive the error goes away under a minute
